lib/BackbeatConsumer.js

```diff
@@ -149,7 +149,7 @@ class BackbeatConsumer extends EventEmitter {
 
     _incrementMetrics(entry) {
         const qEntry = QueueEntry.createFromKafkaEntry(entry);
-        if (!qEntry.error) {
+        if (!qEntry.error && qEntry.constructor.name === 'ObjectQueueEntry') {
```

Comment:
a more standard way to do the same (plus supports inheritance) is `qEntry instanceof ObjectQueueEntry`

Reply:
Ah cool, this way seemed a bit hacky to me so ty, will change this